packages/venia-ui/lib/components/SavedPaymentsPage/savedPaymentsPage.js

```diff
@@ -0,0 +1,67 @@
+import React, { useMemo } from 'react';
+import { PlusSquare } from 'react-feather';
+
+import { useSavedPaymentsPage } from '@magento/peregrine/lib/talons/SavedPaymentsPage/useSavedPaymentsPage';
+import { mergeClasses } from '@magento/venia-ui/lib/classify';
+
+import Icon from '../Icon';
+import LinkButton from '../LinkButton';
+import { Title } from '../Head';
+
+import { GET_SAVED_PAYMENTS_QUERY } from './savedPaymentsPage.gql';
+import defaultClasses from './savedPaymentsPage.css';
+
+const PAGE_TITLE = 'Saved Payments';
+const SUB_HEADING =
+    'Credit Cards saved here will be available during checkout.';
```

Comment:
Is there a pattern for this? If we're doing i18n in the talon we might want to pass text in talon args and get the translated string results. I imagine something like this:

```js
const strings = new Map([
    ['GREETING', 'Hello, {name}!']
]);

const MyComponent = props => {
  const talonProps = useMyComponent({
    strings
  });

  const {
      translatedStrings
  } = talonProps;

  return <div>{translatedStrings.get('GREETING')}</div>
};
```

Reply:
Translation is new, so the pattern here is still pretty nascent. [`AccountMenuItems`](https://github.com/magento/pwa-studio/blob/develop/packages/venia-ui/lib/components/AccountMenu/accountMenuItems.js#L15-L25) has an early example.

Basically, we define static strings in the [corresponding talon](https://github.com/magento/pwa-studio/blob/f96f8a9abc92b6d89c638ebb1c7207a61346607b/packages/peregrine/lib/talons/AccountMenu/useAccountMenuItems.js#L17-L53), then map them to `FormattedMessage` elements in the component. Notice that the talon doesn't actually perform the translation—`FormattedMessage` does—so it doesn't need to receive strings. This encourages people to replace messages in the JSON file rather than create a whole new component just to call the talon with different strings.

Reply:
OK cool, I see how `FormattedMessage` works.

But from what I can tell, for this component we would just be us returning `ids` from the talon. The account menu items list has other data such as `/url` and `name` but since this is just a few strings in the page I'm not sure a list or map is warranted. I've pushed an intermediate solution which I feel may suffice for the current needs. Let me know what you think.

https://github.com/magento/pwa-studio/pull/2671/commits/3ee5d9c3030620489319d5248e666ffe8a2d8771